README.md

```diff
@@ -15,6 +15,10 @@ Slack app is based on App Template - you can check it [here](https://github.com/
 
 ## How to use this project
 
+### Create Upstash account
+
+You need [Upstash](https://upstash.com/) to run this application locally. Follow [How to configure Upstash for local development](docs/upstash.md) for more info.
```

Comment:
Discussed with @lkostrowski approach we want to have regarding APLs:

- add APL env variable, with default string `FileAPL`
- modify `saleor-app.ts` file to choose APL based on `APL` variable. Available choices: FileAPL, VercelAPL, UpstashAPL
- readme should contain information, that local development can be done with FileAPL (works without any dependencies). If you want to deploy or have multitenancy, use UpstashAPL.
- VercelAPL will probably be only used by the Marketplace, so don't think we have to describe it in this repository